hivemind/utils/logging.py

```diff
@@ -27,9 +66,9 @@ def get_logger(module_name: str) -> logging.Logger:
 
     logging.addLevelName(logging.WARNING, "WARN")
     formatter = CustomFormatter(
-        fmt="[{asctime}.{msecs:03.0f}][{levelname}][{caller}] {message}",
+        fmt="{asctime}.{msecs:03.0f}  {bold}{levelcolor}{levelname}{reset} {bold}{caller}{reset}  {message}",
```

Comment:
I'd rather keep the square brackets for simplicity of log parsing

Reply:
We've agreed on this format:

![Screen Shot 2021-09-02 at 5 47 21 PM](https://user-images.githubusercontent.com/8748943/131867423-3afa3a18-fba7-4e4e-b74d-61b7eb6db317.png)